<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="tests" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="tests" name="Test(failed)">
    <classes>
      <class name="tests.RegisterUserFresh">
        <methods>
          <include name="tearDown"/>
          <include name="setUp"/>
          <include name="userRegister"/>
        </methods>
      </class> <!-- tests.RegisterUserFresh -->
      <class name="tests.LoginUser">
        <methods>
          <include name="Login" invocation-numbers="0"/>
          <include name="setUp"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- tests.LoginUser -->
      <class name="tests.Failintentional">
        <methods>
          <include name="failedTest"/>
          <include name="tearDown"/>
        </methods>
      </class> <!-- tests.Failintentional -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
